src/InteractiveTutorial.js

```diff
@@ -1,55 +1,77 @@
 import {Engine}                         from "./engine";
 import {Player}                         from "./Player";
-import {dialogBoxCreate}                from "../utils/DialogBox";
+import {Settings}                       from "./Settings";
+import {Terminal}                       from "./Terminal";
```

Comment:
oy...more circular dependencies....

Reply:
I've removed this (in a local WIP branch)  since it wasn't actually needed